src/content/developers/docs/consensus-mechanisms/pos/keys/index.md

```diff
@@ -0,0 +1,89 @@
+
+- Get the validator slashed) by:
+  - Being a proposer and signing two different beacon blocks for the same slot
+  - Being an attester and signing an attestation that "surrounds" another one.
+  - Being an attester and signing two different attestations having the same target.
+- Force a voluntary exit, which stops the validator from staking, and grants access to its ETH balance to the withdrawal key owner.
+
+The **validator public key** is included in the _deposit data_ which allows Ethereum to identify the validator.
+
+### The withdrawal key {#withdrawal-key}
+
+The withdrawal key will be required to move the validator balance after this is enabled in the upcoming Shanghai upgrade. Just like the validator keys, the withdrawal keys also consist of two components:
+
+- Withdrawal **private** key
+- Withdrawal **public** key
+
+Losing this key means losing access to the validator balance. However, the validator can still sign attestations and blocks since these actions require the validator private key, but there is little to no incentive to do so if the keys are lost.
+
+Separating out the validator keys from the Ethereum account keys enables multiple validators to be run by a single user.
+
+![validator key schematic](validator-key-schematic.png)
+
+## Deriving keys from other keys {#deriving-keys-from-keys}
+
+If every 32 ETH staked required a new set of 2 completely independent keys to be created, key management would quickly become unwieldy especially for users running multiple validators. To get around this problem, multiple validator keys can be derived from a single common secret. Storing that single secret allows access to multiple validator keys.
+
+[Mnemonics](https://en.bitcoinwiki.org/wiki/Mnemonic_phrase) and paths are a well-known feature that users often encounter when [they access](https://ethereum.stackexchange.com/questions/19055/what-is-the-difference-between-m-44-60-0-0-and-m-44-60-0) their hardware wallets. The mnemonic is a sequence of words that act as an initial seed for a private key. When combined with additional data, the mnemonic is used to generate a hash known as the 'master key'. This can be thought of as the root of a tree. Branches from this root can then be drived using a heirarchical path, so that child nodes can exist as combinations of their parent node's hash and their index in the tree.
+
+These paths have the following structure which will be familiar to users who have interacted with hardware wallets:
+
+```
+m/44'/60'/0'/0`
+```
+
+The slashes in this path separate components of the private key as follows:
+
+```
+master_key / purpose / coin_type / account / change / address_index
+```
+
+This logic enables a user to attach as many validators to a single **withdrawal key** as they want, because the root of the tree can be common and differentiation can happen at the branches. The user can **derive any number of keys** from the Mnemonic phrase.
```

Comment:
```suggestion
This logic enables users to attach as many validators to a single **withdrawal key** as possible because the tree root can be common, and differentiation can happen at the branches. The user can **derive any number of keys** from the Mnemonic phrase.
```

Reply:
Agree with Josh's suggestions, but also, I'm not sure I understand the way this is worded.

My understanding is each validator has its own set of priv/pub signing keys + priv/pub withdrawal keys. For example, two validators would need two different withdrawal keys to eventually withdraw funds. Both of these can be derived from the seed, but are still unique.

But, with how it's worded here, seems like you're saying a single withdrawal key can be shared... I assume you're referring to a key higher up in the tree, but this threw me off a little. May be nitpicking, but should we maybe refer to this as the "mnemonic phrase" to be consistent with the image?

```suggestion
This logic enables users to attach as many validators as possible to a single **mnemonic phrase** because the tree root can be common, and differentiation can happen at the branches. The user can **derive any number of keys** from the mnemonic phrase.
```